src/Basic/NaturalTransformation.lidr

```diff
@@ -159,3 +160,72 @@ The code above is everything we need to define what a natural transformation is.
 >                             (mapObj fun2 b)
 >                             (mapMor fun1 a b f)
 >                             (component b)
+>
+> naturalTransformationExt :
+>   (cat1, cat2 : Category)
+>   -> (fun1, fun2 : CFunctor cat1 cat2)
+>   -> (trans1, trans2 : NaturalTransformation cat1 cat2 fun1 fun2)
+>   -> ((a : obj cat1) -> component trans1 a = component trans2 a)
+>   -> trans1 = trans2
+> naturalTransformationExt cat1 cat2 fun1 fun2 trans1 trans2 eq = really_believe_me()
```

Comment:
please add a white space between `really_believe_me` and `()`

Reply:
if you want, since you are not using them, you can also use `_` for the input variables